lua/wikis/commons/Patches/PatchList.lua

```diff
@@ -0,0 +1,350 @@
+				ConditionNode(ColumnName('date'), Comparator.eq, endDate)
+			}
+		}
+	end
+
+	return mw.ext.LiquipediaDB.lpdb('datapoint', {
+		conditions = conditions:toString(),
+		order = 'date desc, pageid desc',
+		limit = math.min(tonumber(args.limit) or 100, 5000)
+	})
+end
+
+--- Formats an array of highlights into a wiki-compatible list
+--- @param arr table Array of highlight strings
+--- @return table Array of formatted list items
+function PatchList._makeList(arr)
+	local result = {}
+	for _, value in ipairs(arr) do
+		if String.isNotEmpty(value) then
+			table.insert(result, '\n* ' .. value .. '\n')
+		end
+	end
+	return result
+end
+
+--- Builds the table header based on the columns table
+--- @param columns table The list of columns to include in the header
+--- @return string The constructed header row HTML
+local function buildHeader(columns)
+	local header = mw.html.create('tr')
+	for _, column in ipairs(columns) do
+		header:tag('th')
+			:cssText(column.style)
+			:css('font-size', '16px')
+			:wikitext(column.title)
+	end
+	return header
+end
+
+--- Builds a row for a single patch
+--- @param patch table The patch data
+--- @param columns table The list of columns to display
+--- @return string The constructed row HTML
+local function buildRow(patch, columns)
+	local row = mw.html.create('tr')
+	for _, column in ipairs(columns) do
+		row:tag('td')
+			:cssText(column.style)
+			:css('font-size', '14px')
+			:wikitext(column.func(patch))
+	end
+	return row
+end
+
+--- Builds a month header row
+--- @param monthAnchor string The anchor for the month
+--- @param monthAbbr string The abbreviated month name
+--- @param columnCount integer The number of columns in the table
+--- @return string The constructed month header row HTML
+local function buildMonthHeader(monthAnchor, monthAbbr, columnCount)
+	local subheaderRow = mw.html.create('tr')
+	subheaderRow:tag('td')
+		:attr('colspan', columnCount)
+		:addClass('gray-bg')
+		:css('text-align', 'center')
+		:wikitext(
+		'<span id="' .. monthAnchor ..
+		'" style="position: relative; top: -250px; visibility: hidden;"></span><b>' .. monthAbbr .. '</b>'
+	)
+	return subheaderRow
+end
+
+--- Finds the latest patch across all tables
+function PatchList.findLatestPatch()
+	local latestPatches = PatchList.getPatches({limit = 1})
+	if #latestPatches > 0 then
+		local latestPatch = latestPatches[1]
+		PatchList.globalLatestPatchDate = latestPatch.date
+		PatchList.globalLatestPatchId = latestPatch.pageid
+	end
+end
+
+--- Creates and returns the patch table display
+--- Entry point for the module
+--- @param frame table The MediaWiki frame object providing arguments
+--- @return string HTML string representing the patch table
+function PatchList.create(frame)
+	local args = Arguments.getArgs(frame)
+
+	-- Configure table classes for collapsibility
+	local tableClasses = 'wikitable mw-collapsible'
+	if Logic.readBool(args.collapsed) then
+		tableClasses = tableClasses .. ' mw-collapsed'
```

Comment:
imho use general collapsible instead of `mw-collapsed` and `mw-collapsible` classes

Reply:
the js it uses is